src/common/meta/src/rpc/store.rs

```diff
@@ -509,6 +509,13 @@ impl BatchDeleteRequest {
         }
     }
 
+    #[inline]
+    /// Sets `keys`.
```

Comment:
This should violate the latest style guide to hold comment above attributions.

And IIRC @killme2008 suggests that we never use `#[inline]` but let the compiler decide it.